### PR TITLE
Avoid too new testcases

### PR DESCRIPTION
**Pull request by tanin47**
Avoid too new testcases because their minimization tasks might not finish yet.

Update `CONTRIBUTING.md`.

### CODE REVIEW

File: ci/continuous_integration/daemon/main.py [hunk 1]
```diff
@@ -124,6 +125,16 @@ def get_supported_jobtypes():
   return result
 
 
+def is_time_valid(time_in_seconds):
+  """Check if the timestamp is valid for running on the CI."""
+  current_time = time.time()
+  return (
+      # The testcase is not too old.
+      (current_time - time_in_seconds) < MAX_AGE and
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: nit: (current_time - time_in_seconds) to a local variable diff_time

File: ci/continuous_integration/daemon/main.py [hunk 2]
```diff
@@ -29,7 +29,8 @@
 BINARY_LOCATION = '/python-daemon-data/clusterfuzz'
 TOOL_SOURCE = os.path.join(HOME, 'clusterfuzz-tools')
 MAX_PREVIEW_LOG_BYTE_COUNT = 100000
-NINETY_DAYS_IN_SECONDS = 90 * 86400
+MAX_AGE = 90 * 86400  # 90 days.
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: nit: 86400 into 24 * 60 * 60

File: ci/continuous_integration/tests/daemon/main_test.py [hunk 3]
```diff
@@ -189,22 +190,17 @@ def setUp(self):
...
-                {'jobType': 'support', 'id': 23456,
-                 'timestamp': main.NINETY_DAYS_IN_SECONDS},
-                {'jobType': 'supported', 'id': 23456,
-                 'timestamp': main.NINETY_DAYS_IN_SECONDS},
-                {'jobType': 'supported', 'id': 1337,
-                 'timestamp': main.NINETY_DAYS_IN_SECONDS},
+                {'jobType': 'supported', 'id': 12345, 'timestamp': 1},
+                {'jobType': 'supported', 'id': 555, 'timestamp': 2.2},
+                {'jobType': 'unsupported', 'id': 98765, 'timestamp': 3},
+                {'jobType': 'support', 'id': 23456, 'timestamp': 4},
```
Comment (inferno-chromium) [inline ci/continuous_integration/tests/daemon/main_test.py]: should support be supported ?

Comment (inferno-chromium) [inline ci/continuous_integration/tests/daemon/main_test.py]: dont understand how this test work, why we are updating is_time_valid

File: ci/continuous_integration/daemon/main.py (hunk 2 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Done

File: ci/continuous_integration/daemon/main.py (hunk 1 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Done

File: ci/continuous_integration/tests/daemon/main_test.py (hunk 3 above)
Comment (tanin47) [inline ci/continuous_integration/tests/daemon/main_test.py]: `is_time_valid` returns `False` if `timestamp` is `2`.

`supported` and `support` are the supported job types in this testcase. I'll rename them to `supported_0` and `supported_1`.

**Pull request merged**